.changeset/quick-pillows-jump.md

```diff
@@ -0,0 +1,15 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": tree
+---
+
+Add `allowUnused` utility
+
+Add an `allowUnused` utility function which discards its type or runtime argument.
+This can be used to enable use of [ValidateRecursiveSchema](https://fluidframework.com/docs/api/fluid-framework/validaterecursiveschema-typealias) in environments where TypeScript is configured to reject code with unused locals.
```

Comment:
```suggestion
When TypeScript is configured to reject code with unused locals, this function can be used so that  [ValidateRecursiveSchema](https://fluidframework.com/docs/api/fluid-framework/validaterecursiveschema-typealias) works correctly.
```

"Works correctly" in my suggestion could probably be improved.

Reply:
```suggestion
When TypeScript is configured to reject code with unused locals, this function can be used to suppress that error enabling use of [ValidateRecursiveSchema](https://fluidframework.com/docs/api/fluid-framework/validaterecursiveschema-typealias) to compile.
```